duckduckgo/duckduckgo-privacy-extension
# Changes to in context signup treatment

**Reviewer:**



Task: https://app.asana.com/0/312629933896096/1204087642219392/f

## Description:
Updates in-context signup

---

Updates Autofill to version [6.4.1](https://github.com/duckduckgo/duckduckgo-autofill/releases/tag/6.4.1).

### Autofill 6.4.0 and 6.4.1 release notes
## What's Changed

### Tooling and Tests
* Update integration tests for in-context signup by @alistairjcbrown in https://github.com/duckduckgo/duckduckgo-autofill/pull/254 and https://github.com/duckduckgo/duckduckgo-autofill/pull/255
* Update macOS browser repository ID in `asana-release.yml` by @ayoy in https://github.com/duckduckgo/duckduckgo-autofill/pull/259
* Use node v18 and update actions to read `.nvmrc` file by @alistairjcbrown in https://github.com/duckduckgo/duckduckgo-autofill/pull/269
* Bugfix: Update release script to add checkout path to node version file path by @alistairjcbrown in https://github.com/duckduckgo/duckduckgo-autofill/pull/286

### Source Code Updates
* Refactor in-context state to feature file by @shakyShane in https://github.com/duckduckgo/duckduckgo-autofill/pull/251
* Prevent duplicated schema types by using JSON schema throughout by @shakyShane in https://github.com/duckduckgo/duckduckgo-autofill/pull/253
* Stop scanning pages with a large number of inputs by @alistairjcbrown in https://github.com/duckduckgo/duckduckgo-autofill/pull/257 and https://github.com/duckduckgo/duckduckgo-autofill/pull/262
* Ignore small email inputs by @alistairjcbrown in https://github.com/duckduckgo/duckduckgo-autofill/pull/261
* Fix username not saved by @GioSensation in https://github.com/duckduckgo/duckduckgo-autofill/pull/275
* Updates to in-context signup treatment by @alistairjcbrown and @GioSensation in https://github.com/duckduckgo/duckduckgo-autofill/pull/284


**Full Changelog**: https://github.com/duckduckgo/duckduckgo-autofill/compare/6.3.0...6.4.1

---

## Steps to test this PR:
1. Install extension
2. Go to https://duckduckgo.com/newsletter
3. Confirm gray Dax icon shows on email input -- clicking Dax icon will show in-context sign-up, clicking on the input will not
4. Sign in to Email Protection
5. Confirm red icon shows -- clicking Dax icon will show Email Protection autofill

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
